docs/framework/additional-apis/index.md

```diff
@@ -1,60 +1,3 @@
-  
-In addition, some libraries target specific platforms or implementations of the .NET Framework. For example, the <xref:System.Text.CodePagesEncodingProvider> class makes code page encodings available to UWP apps developed using the .NET Framework. This topic lists these libraries as well.  
-  
-## OOB projects
-  
-| Project | Description |  
-| ------- | ----------- |  
-| <xref:System.Collections.Immutable> | Provides collections that are thread safe and guaranteed to never change their contents. |
-| <xref:System.Net.Http.WinHttpHandler> | Provides a message handler for <xref:System.Net.Http.HttpClient> based on the WinHTTP interface of Windows. |
-| <xref:System.Numerics> | Provides a library of vector types that can take advantage of SIMD hardware-based acceleration.| 
-| <xref:System.Threading.Tasks.Dataflow> | The TPL Dataflow Library provides dataflow components to help increase the robustness of concurrency-enabled applications. |  
-
-## Platform-specific libraries
-  
-| Project | Description |  
-| ------- | ----------- |  
-| <xref:System.Text.CodePagesEncodingProvider> | Extends the <xref:System.Text.EncodingProvider> class to make code page encodings available to apps that target the Universal Windows Platform. |  
-  
-## Private APIs  
-
-These APIs support the product infrastructure and are not intended/supported to be used directly from your code.  
-  
-| API Name |
-| -------- |
-| [System.Net.Connection Class](connection.md) |
-| [System.Net.Connection.m\_WriteList Field](m_writelist.md) |
-| [System.Net.ConnectionGroup Class](connectiongroup.md) |
-| [System.Net.ConnectionGroup.m\_ConnectionList Field](m_connectionlist.md) |
-| [System.Net.CoreResponseData Class](coreresponsedata.md) |
-| [System.Net.CoreResponseData.m\_ResponseHeaders Field](coreresponsedata_m_responseheaders.md) |
-| [System.Net.CoreResponseData.m\_StatusCode Field](coreresponsedata_m_statuscode.md) |
-| [System.Net.HttpWebRequest.\_AutoRedirects Field](_autoredirects.md) |
-| [System.Net.HttpWebRequest.\_CoreResponse Field](httpwebrequest__coreresponse.md) |
-| [System.Net.HttpWebRequest.\_HttpResponse Field](_httpresponse.md) |
-| [System.Net.ServicePoint.m\_ConnectionGroupList Field](m_connectiongrouplist.md) |
-| [System.Net.ServicePointManager.s\_ServicePointTable Field](s_servicepointtable.md) |
-| [System.Windows.Diagnostics.VisualDiagnostics.s\_isDebuggerCheckDisabledForTestPurposes Field](s-isdebuggercheckdisabledfortestpurposes-field.md) |
-| [System.Windows.Forms.Design.DataMemberFieldEditor Class](datamemberfieldeditor-class.md) |
-| [System.Windows.Forms.Design.DataMemberListEditor Class](datamemberlisteditor-class.md) |
-  
-## See also
-
-- [The .NET Framework and Out-of-Band Releases](../get-started/the-net-framework-and-out-of-band-releases.md)
+redirect_url: microsoft.sqlserver.server.smiorderproperty.item
```

Comment:
I think that this makes for a really confusing experience when using either the breadcrumb (which redirects to the smiorderproperty.item property) and the TOC. I'd consider restoring the page -- it at least makes the basic structure of the API reference section clear -- something lacking when the page is removed.
In any case, you can merge when you're satisfied with the result.